src/visualizations/config/adapters/dhis_highcharts/xAxis/scatter.js

```diff
@@ -12,20 +12,26 @@ import { getAxis } from '../../../../util/axes'
 import {
     getGridLineColor,
     getLabels,
-    getMaxValue,
     getMinValue,
     getRegressionLine,
 } from '../axis'
+import { PROP_EXTREME_LINES } from '../../../../../modules/outliers'
 
 const AXIS_TYPE = 'RANGE'
 const AXIS_INDEX = 1
 
-export default function (layout, series) {
+export default function (layout, series, extraOptions) {
     const dataValues = series?.map(item => item.data).flat()
     const axis = getAxis(layout.axes, AXIS_TYPE, AXIS_INDEX)
+    const extremeObj = extraOptions.outlierHelper?.vars?.config?.extremeLines
+        ?.enabled
+        ? extraOptions.outlierHelper[PROP_EXTREME_LINES][0]
+        : null
+    const xMax = extraOptions.outlierHelper?.vars?.xyStats?.xMax
+
     return objectClean({
         min: getMinValue(axis.minValue, dataValues),
-        max: getMaxValue(axis.maxValue, dataValues),
+        max: extremeObj?.value > xMax ? extremeObj?.value * 1.1 : undefined,
```

Comment:
Should this fallback to `getMaxValue` instead of `undefined`?

```suggestion
        max: extremeObj?.value > xMax ? extremeObj?.value * 1.1 : getMaxValue(axis.maxValue, dataValues),
```